cmd/quadlet/main.go

```diff
@@ -303,7 +311,22 @@ func warnIfAmbiguousName(container *parser.UnitFile) {
 }
 
 func main() {
+	var prevError error
```

Comment:
I am not sure I get what you are trying to achieve with `prevError`. It seems that you are trying to avoid printing the last error twice. If so, wouldn't it be simpler not to log anything inside `main` (along with `reportError` and the deferred function) and have the logs printed where they occur? If this isn't the case, can you please explain?